### ISSUE TITLE
Print tables in parallel 

### ISSUE THREAD

**Issue by patrick-hermann-sva** [jedib0t/go-pretty#244]
i got some cool stuff printed out with your tables in various cli projects, thank you very much for your work @jedib0t!

is there a way to print those two (or even three) whole tables in parallel? obviously i print those independent tables from their own data source (retrieve the information for each by k8s api call) in a loop row by row but i would be nice to have the tables print next to each other for no need to scroll..

```
? select (type): namespace
? select (cluster): barrybonds, teasle
┌─────────────┬─────────────────────────────┐
│ CLUSTERNAME │ NAMESPACES                  │
├─────────────┼─────────────────────────────┤
│ barrybonds  │ calico-system               │
│             │ cattle-fleet-system         │
│             │ cattle-impersonation-system │
│             │ cattle-system               │
│             │ default                     │
│             │ kube-node-lease             │
│             │ kube-public                 │
│             │ kube-system                 │
│             │ local                       │
│             │ longhorn-system             │
│             │ minio-operator              │
│             │ tigera-operator             │
└─────────────┴─────────────────────────────┘
┌─────────────┬─────────────────────────────┐
│ CLUSTERNAME │ NAMESPACES                  │
├─────────────┼─────────────────────────────┤
│ teasle      │ argo-cd                     │
│             │ calico-system               │
│             │ cattle-fleet-system         │
│             │ cattle-impersonation-system │
│             │ cattle-system               │
│             │ default                     │
│             │ ingress-nginx               │
│             │ kube-node-lease             │
│             │ kube-public                 │
│             │ kube-system                 │
│             │ local                       │
│             │ metallb-system              │
│             │ sthings-runner              │
│             │ sthings-tekton              │
│             │ tekton-pipelines            │
│             │ tigera-operator             │
│             │ trident                     │
└─────────────┴───────────────────────────
```

**Comment by jedib0t** [comment]
Hey! Not sure if I understand the request here... you can do this by using multiple table objects already... so I'm a little confused. 

Can you explain?

**Comment by jedib0t** [comment]
Closing due to lack of response. Feel free to re-open if you want to with more information.
